sources/shared-defaults/public-host-containers.toml

```
@@ -6,9 +6,17 @@
 [settings.host-containers.admin]
 enabled = false
 superpowered = true
-source = "public.ecr.aws/bottlerocket/bottlerocket-admin:v0.11.14"
+
+[metadata.settings.host-containers.admin.source.setting-generator]
+command = "schnauzer-v2 render --template 'public.ecr.aws/bottlerocket/bottlerocket-admin:v0.11.14'"
+strength = "weak"
+skip-if-populated = true
```

Since these are static strings and don't require templates, can we disregard the setting generator and accomplish the same thing like this?

```suggestion
[settings.host-containers.admin]
enabled = false
superpowered = true
source = "public.ecr.aws/bottlerocket/bottlerocket-admin:v0.11.14"

[metadata.settings.host-containers.admin.source]
strength = "weak"
```